src/Backend/Request.php

```diff
@@ -25,21 +24,6 @@ public function __construct(ClientInterface $client)
     }
 
     /**
-     * @param string $url
-     * @param string $method
-     * @param array  $options
-     *
-     * @return RequestInterface
-     */
-    protected function createRequest($url, $method = 'GET', $options = [])
```

Comment:
Seems that we have different views on that. Making things private may cause you real headaches in unittests. So my opinion is that protected does not automatically mean "public api".

2 options:
- create the function again and omit the options (which haven't been working anyway)
- create a new major release noting this BC.